src/web/app/components/question-types/question-edit-details-form/question-edit-details-form.component.ts

```diff
@@ -38,4 +38,33 @@ export abstract class QuestionEditDetailsFormComponent<D extends FeedbackQuestio
   triggerModelChangeBatch(obj: Partial<D>): void {
     this.detailsChange.emit({ ...this.model, ...obj });
   }
+
+  onPointsInput(event: KeyboardEvent): void {
```

Comment:
Since we're using this method for the max and min value inputs in numeric scale, I think this method requires a more general name. Perhaps `onIntegerInput`?